Assignment4/src/main/java/com/example/Assignment4/controller/ItemController.java

```diff
@@ -0,0 +1,68 @@
+package com.example.Assignment4.controller;
+
+import com.example.Assignment4.entity.Items;
+import com.example.Assignment4.service.ItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+import java.util.List;
+
+/*
+ * It is use as rest controller
+ */
+@RestController
+public class ItemController {
+
+    @Autowired
+    public ItemService itemService;
+
+    /*
+     * Welocome page of server
+     */
+    @GetMapping("/home")
```

Comment:
is this  required ? if not don't push it.

Reply:
ok